app/models/va_profile_redis/v2/contact_information.rb

```diff
@@ -179,6 +179,10 @@ def response_from_redis_or_service
       def contact_info_service
         @service ||= VAProfile::V2::ContactInformation::Service.new @user
       end
+
```

Comment:
Users with an ICN are verified users. Only verified users are permitted to use profile services. VAProfileRedis::V2::ContactInformation should only obtain data from verified users.